bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/cloudConnectInverter.xml

```diff
@@ -0,0 +1,115 @@
+			</channel>
+			<channel id="pv3-power" typeId="system.electric-power">
+				<label>PV 3 Power</label>
+				<description>Electric power of PV String 3</description>
+			</channel>
+			<channel id="pv4-power" typeId="system.electric-power">
+				<label>PV 4 Power</label>
+				<description>Electric power of PV String 4</description>
+			</channel>
+			<channel id="pv-total-power" typeId="system.electric-power">
+				<label>PV Total Power</label>
+				<description>The sum of PV powers from all PV strings</description>
+			</channel>
+			<channel id="battery-level" typeId="system.battery-level">
+				<label>Battery Level</label>
+				<description>The battery state of charge in percent</description>
+			</channel>
+			<channel id="battery-power" typeId="system.electric-power">
+				<label>Battery Power</label>
+				<description>Power to/from the battery</description>
+			</channel>
+			<channel id="feed-in-power" typeId="system.electric-power">
+				<label>Feed-In Power</label>
+				<description>Power to/from the electricity network.</description>
+			</channel>
+			<channel id="total-feed-in-energy" typeId="system.electric-energy">
+				<label>Total Feed-In Energy</label>
+				<description>Total energy feed-in to the electricity network.</description>
+			</channel>
+			<channel id="total-consumption" typeId="system.electric-energy">
+				<label>Total Consumed Energy</label>
+				<description>Total Energy consumed from the electricity network.</description>
+			</channel>
+			<channel id="inverter-meter2-power" typeId="system.electric-power">
+				<label>Meter2 Power</label>
+				<description>Inverter power on meter2.</description>
+			</channel>
+			<channel id="inverter-eps-power-r" typeId="system.electric-power">
+				<label>EPS power R</label>
+				<description>Inverter AC EPS power R</description>
+			</channel>
+			<channel id="inverter-eps-power-s" typeId="system.electric-power">
+				<label>EPS power S</label>
+				<description>Inverter AC EPS power S</description>
+			</channel>
+			<channel id="inverter-eps-power-t" typeId="system.electric-power">
+				<label>EPS power T</label>
+				<description>Inverter AC EPS power T</description>
+			</channel>
+
+			<channel id="today-energy" typeId="system.electric-energy">
+				<label>Yield today</label>
+				<description>Inverter output energy for the day</description>
+			</channel>
+			<channel id="total-energy" typeId="system.electric-energy">
+				<label>Yield total</label>
+				<description>Total inverter output energy</description>
+			</channel>
+
+			<channel id="last-update-time" typeId="last-retrieve-time-stamp"/>
+
+			<channel id="inverter-workmode" typeId="inverter-workmode">
+				<label>Inverter Workmode</label>
+				<description>Inverter Workmode</description>
+			</channel>
+
+			<channel id="raw-data" typeId="raw-data-type"/>
+		</channels>
+
+		<properties>
+			<property name="thingTypeVersion">1</property>
+		</properties>
```

Comment:
Don't think this is needed as this is the first version if this type structure and no thing upgrade instructions are written for that first version.